File: content/developers/docs/api/snippet/auth-byname.ts
const api = new Jellyfin(/* .. */);

try {
  // highlight-start
  const authenticationResult = await api.authenticateUserByName('demo', '');
  // highlight-end

  // Log session information
  console.log(authenticationResult.data.sessionInfo);
} catch (err) {
  // TODO: Check how to catch 401 with TS SDK
  if (err.status == 401) {
    // Username or password is incorrect
    console.log('Invalid user');
  }
}

File: content/developers/docs/api/snippet/auth-byname.kt
val api = jellyfin.createApi(/* .. */)

try {
// highlight-start
	val authenticationResult by api.userApi.authenticateUserByName(
		username = "demo",
		password = "",
	)
// highlight-end

	// Use access token in api instance
	api.accessToken = authenticationResult.accessToken

	// Print session information
	println(authenticationResult.sessionInfo)
} catch (err: InvalidStatusException) {
	if (err.status == 401) {
		// Username or password is incorrect
		println("Invalid user")
	}
}
